### PR TITLE
Stoppable process.

### PR DESCRIPTION
**Pull request by kubaplas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no

Added status and stoppable columns to processes table, also to all classes connected to Process model.
Status is displayed in the admin panel as an additional column. If the process is running and has stoppable flag set to true, button will show up in admin panel, which would change the status of the process to "stopping".
This way processes can detect if the user wants to stop given process and do so.

### CODE REVIEW

File: src/ProcessManagerBundle/Controller/ExecutableController.php [hunk 1]
```diff
@@ -21,6 +21,13 @@
 
 class ExecutableController extends ResourceController
 {
+    public function getAction(Request $request)
+    {
+        $page = $request->get('page');
```
Comment (dpfaffenbauer) [inline src/ProcessManagerBundle/Controller/ExecutableController.php]: whats the point in that?

File: src/ProcessManagerBundle/Resources/translations/admin.en.yml [hunk 2]
```diff
@@ -17,3 +17,10 @@ processmanager_log_download: 'Log Download'
 processmanager_artifact_download: 'Artifact Download'
 processmanager_started: 'Started'
 processmanager_completed: 'Completed'
+processmanager_failed: 'Failed'
+processmanager_completed_with_exceptions: 'Competed (!)'
```
Comment (dpfaffenbauer) [inline src/ProcessManagerBundle/Resources/translations/admin.en.yml]: typo

**Review by dpfaffenbauer — CHANGES_REQUESTED** [review]

File: src/ProcessManagerBundle/Controller/ExecutableController.php (hunk 1 above)
Comment (kubaplas) [reply src/ProcessManagerBundle/Controller/ExecutableController.php]: I removed the code.

**Comment by dpfaffenbauer** [comment]
Good, the workflow is that PM only sets the status, the process it self needs to check if the status has been changed right?

**Comment by kubaplas** [comment]
Exactly. I have PoC ready for Import Defitnions here: https://github.com/w-vision/DataDefinitions/pull/246nd=1

File: README.md [hunk 3]
```diff
@@ -104,6 +104,29 @@ Import Definitions has an example implementation of that [Import Definition Repo
...
+```
+$process->setStoppable(true);
+$process->setStatus(ProcessManagerBundle::STATUS_RUNNING);
+$process->save();
+```
+
+Additionally, you need to implement stop logic to your process. Track the process status and stop your process if it's set to `ProcessManagerBundle::STATUS_STOPPING`:
+
+```php
+$process = $this->processRepository->findOneBy(['id' => $processId]);
```
Comment (dpfaffenbauer) [inline README.md]: `find($processId)`

**Review by dpfaffenbauer — CHANGES_REQUESTED** [review]

**Comment by dpfaffenbauer** [comment]
I think other than my comment, it seems fine

File: README.md (hunk 3 above)
Comment (kubaplas) [reply README.md]: Changed.

**Comment by dpfaffenbauer** [comment]
thanks a lot, again :)

**Pull request merged**